app/views/layouts/_header.html.slim

```diff
@@ -50,4 +50,5 @@ header
             - else
               = render 'layouts/logged_content'
         - if current_user.banned?
-          div.banned Ваш обліковий запис заблоковано!
+          div.banned
+            = "Ваш обліковий запис заблоковано! Причина: #{current_user.ban_reason}"
```

Comment:
I think it will be better to move div.banned in application.html.slim file, right after <main> tag